BACKEND/userAuth/src/main/java/com/apptime/auth/repository/TaskRepository.java

```diff
@@ -23,7 +23,7 @@
 	Task findByUserNameAndState(String userName, TaskState state);
 
 	Task deleteById(long id);
-
-	@Query("select u from Task u where u.scheduledstart >= :start and u.userName = :name")
-    Set<Task> getTasksStartedLaterThan( Date start,String name);
+	//@Query( value = "SELECT * FROM   task WHERE  (scheduledstart AT TIME ZONE 'CDT')::date = :start and username = :name", nativeQuery = true)
```

Comment:
We should remove this line.

Reply:
ops! forgot to remove this commented code.